modules/apps/27-interchain-accounts/types/errors.go

```diff
@@ -4,6 +4,12 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
+const (
+	// AcknowledgementError defines a string constant included in error acknowledgements
+	// NOTE: Changing this const is state machine breaking as acknowledgements are written into state
+	AcknowledgementError = "error handling packet on destination chain: see events for details"
```

Comment:
Named the const as `AcknowledgementError`, didn't want to cause any friction between actual sdk registered errors below.